src/axolotl/utils/schemas/peft.py

```diff
@@ -41,7 +41,7 @@ class LoraConfig(BaseModel):
     adapter: str | None = Field(
         default=None,
         json_schema_extra={
-            "description": "If you want to use 'lora' or 'qlora' or leave blank to train all parameters in original model"
+            "description": "If you want to use 'lora' or 'qlora' or 'qalora' or leave blank to train all parameters in original model"
         },
     )
```

Comment:
Although unrelated, but I wonder if we can set `adapter: AdapterEnum | None`. This should simplify our checks later below to not allow incorrect string.